site/dat/repo/various.json

```diff
@@ -1313,5 +1313,25 @@
             "downloadUrl": "https://github.com/rollno748/di-kafkameter/releases/download/1.0/di-kafkameter-1.0.jar"
         }
     }
-}
+  },
+  {
+    "id": "jmeter-schema-assertion",
```

Comment:
The prefix `jmeter-` is confusing with core IMO. Either remove it or replace with your own prefix.